src/tests/functional/plugin/gpu/shared_tests_instances/multi/gpu_remote_blob_tests.cpp

```diff
@@ -8,20 +8,29 @@
 #include "multi/multi_remote_blob_tests.hpp"
 #include "multi/multi_remote_blob_multidevice_test.hpp"
 #include "common_test_utils/test_constants.hpp"
+#include <openvino/runtime/auto/properties.hpp>
+
+using MultiDevice_Bind_test = MultiDevice_Test;
 
 auto device_names_and_support_for_remote_blobs = []() {
-    return std::vector<DevicesNamesAndSupportPair>{
-        {{GPU}, true},      // GPU via MULTI,
-        {{"GPU.0"}, true},  // GPU.0 via MULTI,
+    return std::vector<DevicesNamesAndSupportTuple>{
+        {{GPU}, true, {}},      // GPU via MULTI,
+        {{"GPU.0"}, true, {}},  // GPU.0 via MULTI,
+        {{GPU}, true, {ov::intel_auto::device_bind_buffer(true)}},      // GPU via MULTI,
+        {{"GPU.0"}, true, {ov::intel_auto::device_bind_buffer(true)}},  // GPU.0 via MULTI,
 #ifdef ENABLE_INTEL_CPU
-        {{GPU, CPU}, true},  // GPU+CPU
-        {{CPU, GPU}, true},  // CPU+GPU
+        {{GPU, CPU}, true, {}},  // GPU+CPU
+        {{CPU, GPU}, true, {}},  // CPU+GPU
+        {{GPU, CPU}, true, {ov::intel_auto::device_bind_buffer(true)}},  // GPU+CPU
+        //{{CPU, GPU}, true, {ov::intel_auto::device_bind_buffer(true)}},  // CPU+GPU hang
```

Comment:
why there is hang?

Reply:
CPU+GPU hangs when infer request waits for result. it Hangs in src/tests/functional/plugin/shared/include/multi/multi_remote_blob_tests.hpp line 36

Reply:
we have to understand the problem and resolve it

Reply:
disccuss offline